### DISCUSSION TITLE
Welcome to attractplus Discussions!

### DISCUSSION THREAD

**Discussion by oomek** [oomek/attractplus#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mahuti** [comment]
OK. I'm here to listen and provide constructive criticism (argue)

**Comment by oomek** [comment]
That's something I discovered today. I hope it becomes useful.

**Comment by oomek** [comment]
Feel free to edit the welcome message if it feels too generic.

**Comment by substring** [comment]
Suggestion 1 : get rid of MacOS support
Suggestion 2 : get rid of Windows XP support
suggestion 3 : get rid of anything 32bits (except for ARM)

**Reply by oomek** [reply]
2 & 3 seem reasonable. 3 needs a fix of nvapi bug on x64 with nvidia ( still unable to reproduce it on my machine ) 
Regarding 1, as a pc supremacist I claim that mac is not meant for gaiming, but still a lot of people will become unhappy.

**Reply by mahuti** [reply]
As a Mac user of many years, the lack of software was more of a problem for gaming, than my lack of interest or its general capacity. That said, the last time I used any of my Macs for gaming was back in the MacMAME days in like... 2001. It's definitely not designed for gaming, and I've never *really* used them for that. 

Zpaolo11x uses a Mac. He's done some good work, and ArcadeFlow makes a lot of people excited. For that reason alone it's good to keep the Mac build alive if possible. It'd be a shame to lose him. I think Verion is on a Mac as well, and while less frequent with his contributions, a lot of people are really excited by his designs. Even though he's no developer, he periodically works with other folks to get stuff built, and it's usually unique and interesting.

**Comment by spilinek** [comment]
Hi. I just did a pull request for attract to fix a video related crash. Should I submit the same type of fix for attractplus? Are you already incorporating fixes from attract?

**Reply by oomek** [reply]
No worries, I'm going to pull your commit into attractplus. I wonder though what conditions have to be met to reproduce such crashes, because neither me, nor anyone on our discord server complained about it.

**Reply by spilinek** [reply]
That's cool. Could I get an invite to the discord server?

**Reply by oomek** [reply]
Sure, here's the link.
https://discord.gg/h5aK8fQu

**Comment by Wenzon-AV** [comment]
hi oomek.
I realized that there is no logo for attractplus yet.

If you don't mind, I took the liberty of building a new one. If you like it, I'll send you the PSD file (Photoshop).

<a href="https://ibb.co/DzCvpHb"><img src="https://i.ibb.co/PFhPQvY/attractplus.png" alt="attractplus" border="0"></a>

**Reply by oomek** [reply]
Thanks for your contribution, but you are few years late unfortunately. We already have a logo.
![AMP-Icon256](https://github.com/oomek/attractplus/assets/2974860/0af5b582-a669-4dee-8640-566adb4627a3)

**Reply by Wenzon-AV** [reply]
Ok, I couldn't find the attractplus logo anywhere.   Now I will keep it in my stuff. Thanks for responding.

**Reply by oomek** [reply]
You're welcome. As a matter if fact we do need a new intro ;)

**Comment by molekulassrg** [comment]
Hello, I'm here because I'm a retro fanatic and after going through all the frontends I can say that I always liked this one the most since it doesn't limit emulators or platforms and has more modification options since AM and now AM+, with patience I managed to learn how to handle it (although I still have a long way to go) and I really congratulate you for the great work and effort to continue updating it, I have a question that I always suffered with and found little information, can it really be made portable? if so, what is the correct way to write the paths in the configuration file, I have seen ..\ ; \ ; etc. but they are only parts and not a base structure of how it should be to achieve it, in emulators I have been able to do it but once integrated and move it to another PC for testing I have not yet achieved it, it could also be a suggestion to be able to count on this for the development of projects, I thank you in advance for your attention, greetings.
